app/models/manageiq/providers/openstack/cloud_manager/vm/operations/relocation.rb

```diff
@@ -5,6 +5,10 @@ module ManageIQ::Providers::Openstack::CloudManager::Vm::Operations::Relocation
     supports :live_migrate do
       unsupported_reason_add(:live_migrate, unsupported_reason(:control)) unless supports_control?
     end
+    supports :evacuate do
+      msg = validate_vm_control
```

Comment:
dont use `validate_` methods for the new `supports` methods. 
This should use the `supports_control?` method